src/Microsoft.VisualStudio.Services.Agent/AdditionalMaskingRegexes.CredScan.cs

```diff
@@ -74,6 +74,11 @@ public static partial class AdditionalMaskingRegexes
                   @"\b" // pre-match
                 + @"npm_[0-9A-Za-z]{36}" // match
                 + @"\b", // post-match
+
+                  // JWT Tokens
+                  @"\b" // pre-match
+                + @"eyJ[A-Za-z0-9-_]+\.eyJ[A-Za-z0-9-_]+\.[A-Za-z0-9-_.+\/]*" // match
+                + @"\b" // post-match 
```

Comment:
Any regex changes should be gated by a feature flag (off by default)

(As regex changes are risky for perf and functionality)

This is not blocking if the regex is from @michaelcfanning